test/eth.ens.js

```diff
@@ -43,7 +266,7 @@ describe('ens', function () {
             });
         });
 
-        it('should return the owner record for a name', function (done) {
+        it('should return the owner record for a name', async function () {
```

Comment:
Changed to ``async`` function because the ``done`` listener doesn't look clean in this case.